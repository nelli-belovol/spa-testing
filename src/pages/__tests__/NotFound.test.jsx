import { screen, render } from '@testing-library/react';

import { NotFound } from '../NotFound';

describe('NotFound', () => {
  it('should render correctly', () => {
    render(<NotFound />);
    expect(screen.getAllByRole('heading')).toMatchSnapshot();
  });
});
