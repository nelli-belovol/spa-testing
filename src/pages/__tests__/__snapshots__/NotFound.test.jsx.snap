// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render correctly 1`] = `
[
  <h1>
    Hello, from About page
  </h1>,
]
`;

exports[`NotFound should render correctly 1`] = `
[
  <h1>
    Page not found
  </h1>,
]
`;
